src/storage-mover/azext_storage_mover/aaz/latest/storage_mover/_update.py

```diff
@@ -113,7 +132,7 @@ def url(self):
 
         @property
         def method(self):
-            return "PATCH"
```

Comment:
But this command is used to use patch?

Reply:
yes, so to be consistent, should all be generic update. Previously probably generated separately.